doc/dash/dash-sonic-hld.md

```diff
@@ -535,13 +538,14 @@ DASH_ROUTE_TABLE:{{group_id}}:{{prefix}}
 ```
 key                      = DASH_ROUTE_TABLE:group_id:prefix ; Route route table with CA prefix for packet Outbound
 ; field                  = value 
-action_type              = routing_type              ; reference to routing type
+action_type              = routing_type              ; reference to routing type (DEPRECATED)
+routing_type             = routing_type              ; replacement for the deprecated `action_type` field. Must be one of {vnet, vnet_direct, direct, servicetunnel, drop}.
 vnet                     = vnet name                 ; destination vnet name if routing_type is {vnet, vnet_direct}, a vnet other than eni's vnet means vnet peering
 appliance                = appliance id              ; appliance id if routing_type is {appliance} 
 overlay_ip               = ip_address                ; overly_ip to lookup if routing_type is {vnet_direct}, use dst ip from packet if not specified
 overlay_sip_prefix       = ip_prefix                 ; overlay ipv6 src ip if routing_type is {servicetunnel}, transform last 32 bits from packet (src ip)
 overlay_dip_prefix       = ip_prefix                 ; overlay ipv6 dst ip if routing_type is {servicetunnel}, transform last 32 bits from packet (dst ip) 
-underlay_sip             = ip_address                ; underlay ipv4 src ip if routing_type is {servicetunnel,privatelink}; this is the ST GW VIP (for ST traffic) or custom VIP. If specified, overrides pl_underlay_sip from DASH_ENI_TABLE
+underlay_sip             = ip_address                ; underlay ipv4 src ip if routing_type is {servicetunnel} or for DASH_VNET_MAPPING_TABLE entries in this prefix where routing_type is {privatelink}; this is the ST GW VIP (for ST traffic) or custom VIP. If specified, overrides pl_underlay_sip from DASH_ENI_TABLE
```

Comment:
@prsunny can we re-confirm whether this underlay SIP is ever used in PL case.

Reply:
Addressed in this comment: https://github.com/sonic-net/sonic-swss/pull/3048#discussion_r1737183468

Will keep `underlay_sip` as a valid field for PL